### PR TITLE
CurveEditor: fix can't edit right tangent

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by 27thLiz — APPROVED** [review]

File: editor/plugins/curve_editor_plugin.cpp [hunk 1]
```diff
@@ -344,19 +344,19 @@ void CurveEditor::open_context_menu(Vector2 pos) {
 										 _curve_ref->get_point_left_mode(_selected_point) == Curve::TANGENT_LINEAR :
 										 _curve_ref->get_point_right_mode(_selected_point) == Curve::TANGENT_LINEAR;
 
-				_context_menu->set_item_checked(CONTEXT_LINEAR, is_linear);
+				_context_menu->set_item_checked(_context_menu->get_item_count() - 1, is_linear);
```
[the remark this answers is not in the document]
Comment (reduz) [reply editor/plugins/curve_editor_plugin.cpp]: there is get_item_index()

[the remark this answers is not in the document]
Comment (groud) [reply editor/plugins/curve_editor_plugin.cpp]: Message from @reduz : @Noshyaar, could you change the code to get identifier? there is a function in PopupMenu to do it

[the remark this answers is not in the document]
Comment (Noshyaar) [reply editor/plugins/curve_editor_plugin.cpp]: Done. Please review. :)

**Comment by groud** [comment]
LGTM, can merge once travis is done :)

**Pull request merged**